elements/pfe-navigation/src/pfe-navigation.js

```diff
@@ -354,9 +354,13 @@ class PfeNavigation extends PFElement {
         dropdownElement.style.removeProperty("height");
       }
       // Delay aria-hidden so animations can finish
-      window.setTimeout(() => {
-        dropdownElement.setAttribute("aria-hidden", "true");
-      }, 500);
+      // @todo/note: I do not think we need to delay the change of the aria-hidden state since aria-hidden hides the content of the page from screen readers, as soon as the dropdown is closed by the screen reader user the dropdown should get aria-hidden="true" immediately
```

Comment:
I did this because I'm tying the CSS style `visibility:hidden` to it with an attribute selector, but that's probably not a great way to do that.

I'll use a different method to set visibility that we can delay.

Reply:
@wesruv oh okay gotcha, let me know how you end up doing it differently.